packages/dds/sequence/src/test/fuzz/sharedStringOracle.fuzz.spec.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TypedEventEmitter } from "@fluid-internal/client-utils";
+import { createDDSFuzzSuite, type DDSFuzzHarnessEvents } from "@fluid-private/test-dds-utils";
+
+import { SharedStringOracle } from "../../sharedStringOracle.js";
+
+import {
+	baseSharedStringModel,
+	defaultFuzzOptions,
+	SharedStringFuzzFactory,
+	type OracleSharedString,
+} from "./fuzzUtils.js";
+
+const emitter = new TypedEventEmitter<DDSFuzzHarnessEvents>();
+
+emitter.on("clientCreate", (client) => {
+	const channel = client.channel as OracleSharedString;
+	const oracle = new SharedStringOracle(channel);
+
+	channel.oracle = oracle;
+});
+
+export class OracleSharedStringFuzzFactory extends SharedStringFuzzFactory {
+	public close(channel: OracleSharedString) {
```

Comment:
i don't think this is a valid api on the interface, and i don't think it will ever be called.